duckduckgo/Android
# Update copy strings to fix capitalisation

Task/Issue URL: https://app.asana.com/0/488551667048375/1203535193049960/f

### Description
Update copy strings with correct casing

### Steps to test this PR

List of copy changes in https://app.asana.com/0/488551667048375/1203535193049962

_Text color for URL in mac OS screen_
- Install from this branch
- Go to Settings > DuckDuckGo Mac App
- [ ] Check text `duckduckgo.com/mac` color is correct for Light/Dark themes

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20230308_135319_DuckDuckGo](https://user-images.githubusercontent.com/20798495/224091891-83d4c409-e46e-49f5-910e-d8398c0768e3.jpg)|![Screenshot_20230308_135223_DuckDuckGo](https://user-images.githubusercontent.com/20798495/224091879-507631cb-add0-4eeb-b818-b3c833f852ad.jpg)|
![Screenshot_20230308_135312_DuckDuckGo](https://user-images.githubusercontent.com/20798495/224091908-04319a81-20b2-4c87-a239-43ea972f69bf.jpg)|![Screenshot_20230308_135231_DuckDuckGo](https://user-images.githubusercontent.com/20798495/224091936-98d7e700-4dda-488f-b7db-2c2404d304b2.jpg)|
